schemas/tesla_app_schema.js

```diff
@@ -0,0 +1,53 @@
+// MongoDB Schema
+
+user = {
+  "_id": ObjectId("5ddec02bc23c74117af22dac"),
+  "username": "devintyler26",
+  "email": "dtyler.cunningham@gmail.com",
+}
+
+vehicle = {
+  "_id": ObjectId("5678"),
+  "id": 55619287574696016,
+  "vehicle_id": 1154919923,
+  "vin": "5YJ3E1EB5KF386157",
+  "display_name": "Tyl3r",
+  "option_codes": "AD15,...",
+  "color": None,
+  "tokens": [
+    "3f97205e77a67ca7",
+    "08e2f93583d415dc"
+  ],
+  "state": "asleep",
+  "in_service": False,
+  "id_s": "55619287574696016",
+  "calendar_enabled": true,
+  "api_version": 7,
+  "backseat_token": None,
+  "backseat_token_updated_at": None,
+  "user_ids": [ObjectId("5ddec02bc23c74117af22dac"),]
+}
+
+profile = {
```

Comment:
Why isn't this part of the user?

Reply:
I learned to keep users separate from their profiles.
Some examples from stack overflow: 
"A user "owns" various resources on your site, such as comments, etc. If you separate the profile from the user then it's just one more resource. The user is static, while the profile will change from time to time.

Separating it out would also allow you to easily maintain a profile history."

"I would keep it separate. Not all your users would want to fill out a profile, so those would be empty fields sitting in your user table. It also means you can change the profile fields without changing any of the logic of your user model."

Reply:
Until you have a requirement for it, it seems like unneeded complexity.  The point of a NoSQL/document DB is to have jagged data types.

Reply:
True true. Thank you!